src/AsyncWebView.Uno.WinUI/AsyncWebView.Uno.WinUI.csproj

```diff
@@ -0,0 +1,60 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras/3.0.44">
+  <PropertyGroup>
+		<TargetFrameworks>net6.0-windows10.0.18362;netstandard2.0;net6.0-android;net6.0-ios;</TargetFrameworks>
+		<LangVersion>10.0</LangVersion>
+		<!-- Ensures the .xr.xml files are generated in a proper layout folder -->
+		<GenerateLibraryLayout>true</GenerateLibraryLayout>
+		<RootNamespace>AsyncWebView</RootNamespace>
+		<Authors>nventive</Authors>
+		<Company>nventive</Company>
+		<AssemblyName>AsyncWebView.Uno.WinUI</AssemblyName>
+		<PackageId>AsyncWebView.Uno.WinUI</PackageId>
+		<GenerateDocumentationFile>true</GenerateDocumentationFile>
+		<PackageTags>async;webview;mvvm;maui;winui;ios;android</PackageTags>
+		<PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+		<PackageProjectUrl>https://github.com/nventive/AsyncWebView</PackageProjectUrl>
+		<DefineConstants>$(DefineConstants);WINUI</DefineConstants>
+
+		<!-- Needed for Source Link support -->
+		<PublishRepositoryUrl>true</PublishRepositoryUrl>
+		<EmbedUntrackedSources>true</EmbedUntrackedSources>
+		<IncludeSymbols>true</IncludeSymbols>
+		<SymbolPackageFormat>snupkg</SymbolPackageFormat>
+  </PropertyGroup>
+	<PropertyGroup Condition="'$(TargetFramework)'=='netstandard2.0'">
+		<DefineConstants>$(DefineConstants);__WASM__</DefineConstants>
+	</PropertyGroup>
+	<PropertyGroup Condition="$(TargetFramework.Contains('-windows10'))">
```

Comment:
What happens when you remove 10 of -windows10, it should not be limited to Windows10

Reply:
Can you move <DefineConstants>$(DefineConstants);WINDOWS_WINUI</DefineConstants> to the condition at line 37
<When Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'windows'">

Reply:
I see, when we have time, we'll have to fix nventive.view